source/includes/documentation/_material_ui_elements.md

```diff
@@ -3729,19 +3729,20 @@ components based on whether the query matches or not.
     // @FindBy(xpath = "//div[contains(@class, 'MuiTypography-root')]/ancestor::*[contains(@class, 'MuiAlert-root')]")
     @UI("//div[contains(@class, 'MuiTypography-root')]/ancestor::*[contains(@class, 'MuiAlert-root')]")
     public static List<Alert> alertsWithDescription;
-    
+
     @Test(dataProvider = "alertsWithDescriptionTestData", dataProviderClass = AlertDataProvider.class)
-    public void alertsWithDescriptionTest(int alertIndex, String titleText, String messageText) {
+    public void alertsWithDescriptionTest(int alertIndex, String titleText, String messageText, String alertClass) {
```

Comment:
`dataProviderClass = AlertDataProvider.class` is redundant, `alertsWithDescriptionTestData` should be added here to represent data that is used in test